environment.yml

```diff
@@ -28,6 +27,7 @@ dependencies:
   - scipy
   - sphinx
   - pip:
+    - photutils
```

Comment:
I expected this to fail, gonna need to bust out the ``requirements.txt`` again...